dev/com.ibm.ws.microprofile.config.1.1_fat/test-applications/classLoaderCache.war/src/com/ibm/ws/microprofile/appConfig/classLoaderCache/test/ClassLoaderCacheTestServlet.java

```diff
@@ -68,4 +70,24 @@ private static ClassLoader getRootClassLoader() {
         System.out.println("Root ClassLoader: " + rootCL);
         return rootCL;
     }
+
+    private int getConfigCacheSize(ConfigProviderResolver resolver) throws IllegalAccessException, IllegalArgumentException, InvocationTargetException, NoSuchFieldException, SecurityException {
+        int size = -1;
+        try {
+            Method getConfigCacheSizeMethod = resolver.getClass().getMethod("getConfigCacheSize");
+            size = (int) getConfigCacheSizeMethod.invoke(resolver);
+        } catch (NoSuchMethodException e) {
+            Field configsForClassLoaderField = resolver.getClass().getSuperclass().getDeclaredField("configsForClassLoader");
+            configsForClassLoaderField.setAccessible(true);
+            @SuppressWarnings("unchecked")
+            Map<ClassLoader, Config> configsForClassLoader = (Map<ClassLoader, Config>) configsForClassLoaderField.get(resolver);
+            size = configsForClassLoader.size();
+
+            for (Map.Entry<ClassLoader, Config> entry : configsForClassLoader.entrySet()) {
+                System.out.println("CL: " + entry.getKey() + " Config: " + entry.getValue());
+            }
```

Comment:
Did you mean to keep this debug output? If you want to see what was in the cache in the failing case, it would be preferable to extract the whole map and then assert its size with a matcher - this should result in the map contents being included in the failure message.